dockerfiles/compilation/java-6/compile.sh

```diff
@@ -43,14 +43,26 @@ git checkout $SHA1
 
 # Compilation
 
+## Compile and install maven mailetdocs plugin
+mvn clean package install:install -pl mailet/mailetdocs-maven-plugin
+
+## Compile James
 if [ "$SKIPTESTS" = "skipTests" ]; then
    mvn package -DskipTests -Pjpa
 else
    mvn package -Pjpa
 fi
 
-# Retrieve result
-
 if [ $? -eq 0 ]; then
    cp server/app/target/james-server-app-*-app.zip $DESTINATION
 fi
+
+## Generate the mailetdocs
+mvn org.apache.james:mailetdocs-maven-plugin:2.5.1-SNAPSHOT:aggregate
```

Comment:
Wel, that's not so simple; as this project is part of the mailet sub-project and it depends on classes of mailet-api; i need to publish all mailets jars...
I don't know if we expect this yet.